src/Docker.PowerShell/Cmdlets/StartContainer.cs

```diff
@@ -41,5 +72,40 @@ public class StartContainer : MultiContainerOperationCmdlet
         }
 
         #endregion
+
+        internal static async Task StartContainerAsync(
```

Comment:
I think I'd rather that this method end up as a static helper in ContainerOperations rather than have it in one cmdlet but called from another.